.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+name: PHP Compatibility
+on:
+  push:
```

Comment:
Is a branch supposed to be specified here?

Reply:
If we didn't want to limit this action to a branch, the syntax would be `on: push`

Reply:
nope, that means runs on push on every branch

Reply:
I think the syntax is wrong as per [the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-a-single-event)

Reply:
It's a valid syntax but you're right, let's stick with how it's supposed to be. Updated